.circleci/config.yml

```diff
@@ -1,17 +1,26 @@
-version: 2
+version: 2.1
 
+# -------------------------
+#   PARAMETERS
+# -------------------------
+parameters:
+  cache_key_version:
```

Comment:
I don't think this makes sense as a parameter, it is unlikely we're going to want to invalidate all of our caches simultaneously